plugins/services/src/js/reducers/JSONSingleContainerParser.js

```diff
@@ -3,7 +3,6 @@ import {
   JSONParser as constraints
 } from "./serviceForm/JSONReducers/Constraints";
 import { JSONParser as container } from "./serviceForm/Container";
-import { JSONParser as containers } from "./serviceForm/Containers";
```

Comment:
Sorry my `WHAT` was to express why we were importing same file two times but I missed the `s` at the end.